extensions-contrib/opencensus-extensions/src/test/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParserTest.java

```diff
@@ -93,6 +96,8 @@ public void setUp()
             )
         ), null
     );
+
+    config = Mockito.mock(OpenCensusProtobufConfig.class);
```

Comment:
Although I do love Mockito, is mocking really necessary for the `OpenCensusProtobufConfig` class?  It's just a simple data class.  Why not just expose a setter for the `resourceLabelsPrefix` field?

Reply:
I always try not to modify actual code just for testing as we can achieve behaviour by `mocking`. 

Though I have added an instance creator method, as in future there might be other configs hence it will be convenient to add required test config in the `getConfig` method itself.

Reply:
OK fair.

Reply:
agree with @sirianni mocking seems unnecessary. @apoorvmittal10 if you have control over a class, or are writing a new class, it's usually better to structure it so you can test the actual behavior. Otherwise you're only be testing your mocked behavior, which may not reflect reality.